fuel-indexer/src/api.rs

```diff
@@ -60,6 +65,58 @@ pub async fn query_graph(
     }
 }
 
+#[allow(unused_variables)]
+pub async fn health_check(
+    Extension(config): Extension<Arc<IndexerConfig>>,
+    Extension(pool): Extension<IndexerConnectionPool>,
+) -> (StatusCode, Json<Value>) {
+    // Get database status
+    let db_status = pool.is_connected().await.unwrap_or(ServiceStatus::NotOk);
+
+    // Get service uptime
+    let proc = Command::new("ps")
```

Comment:
The uptime could just be some shared data var set when the server starts, instead of having to do a proc command each time